CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 
 * Prepare stub for Storage Indexer. Disable fetching packages from Package Storage v1. [#811](https://github.com/elastic/package-registry/pull/811)
+* Populate fake storage for testing purposes. [#813](https://github.com/elastic/package-registry/pull/813)
```

Comment:
Umm, I am wondering now if these changes need to be mentioned in the changelog, they are not user facing :thinking:

Reply:
Removed.